notebooks/section_3.ipynb

```diff
@@ -7,7 +7,6 @@
    "outputs": [],
```

Comment:
<p>You do not need to call '.legend()' twice, please fix in all figures.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/robustzurcher/analysis/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>